tests/pl/light_HassLightSet.yaml

```diff
@@ -2,31 +2,56 @@ language: pl
 tests:
   # brightness
   - sentences:
-      - "Ustaw jasność sypialni na 50%"
       - "Ustaw jasność w sypialni na 50 procent"
     intent:
       name: "HassLightSet"
       slots:
         brightness: 50
         area: "bedroom"
         name: all
+    response: Ustawiono jasność w sypialni na 50
+  - sentences:
+      - "Ustaw jasność sypialni na 50%"
+    intent:
+      name: "HassLightSet"
+      slots:
+        brightness: 50
+        area: "bedroom"
+        name: all
+    response: Ustawiono jasność w sypialni na 50%
   - sentences:
       - "Zmień jasność lampki nocnej na 50%"
+    intent:
+      name: "HassLightSet"
+      slots:
+        brightness: 50
+        name: light.bedroom_lamp
+    response: Ustawiono jasność lampki nocnej na 50%
+  - sentences:
       - "Ustaw lampkę nocną na 50 procent"
     intent:
       name: "HassLightSet"
       slots:
         brightness: 50
         name: light.bedroom_lamp
+    response: Ustawiono jasność lampkę nocną na 50
   # color
   - sentences:
       - "Zmień kolor lampki nocnej na czerwony"
+    intent:
+      name: "HassLightSet"
+      slots:
+        color: "red"
+        name: "light.bedroom_lamp"
+    response: Ustawiono kolor lampki nocnej na czerwony
+  - sentences:
       - "Ustaw lampkę nocną na czerwono"
     intent:
       name: "HassLightSet"
       slots:
         color: "red"
         name: "light.bedroom_lamp"
+    response: Ustawiono kolor lampkę nocną na czerwono
```

Comment:
Bardzo dziwna odpowiedź, myślę, że powinno być: "Ustawiono kolor lampki nocnej na czerwony" (o ilę się da oczywiście), jeśli nie to może "zmieniono kolor na czerwony"?

Reply:
This is the new problem, how to on fly change the lamp name to correct way. In question we ask `lampkę nocną` but I don't have an idea how to convert it to `lampki nocnej`. What should the algorithm look like? Where should he be? Do you have some proposition?

Reply:
As currently we have no algorith to handle this let's move the discussion to a separate Issue and continue there, leave it for now as is.
https://github.com/Uriziel01/home-assistant-intents-pl/issues/2